decorator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require './nameable'
```

Comment:
- [ ] OPTIONAL
It's recommended to use `require_relative` for classes within a structure and rather use `require` for _gems_ and other packer imports. Kindly endeavor to improve upon this aspect